flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/params/shared/colname/HasKeepColNames.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Can we have some example in `flink-ml-lib/src/test` ? this would be super helpful. 
Especially if one use case of "shared params" can be included.

Reply:
We will provide test case with algorithms later.